ray_data_eval/image_training/image_loader_microbenchmark.py

```diff
@@ -61,7 +61,9 @@ def iterate(dataset, label, batch_size, output_file=None):
         f.write(f"{label},{tput}\n")
 
 
-def build_torch_dataset(root_dir, batch_size, shuffle=False, num_workers=None, transform=None):
+def build_torch_dataset(
```

Comment:
Can you revert these? If Ruff is set to read from `pyproject.toml` and using `line-length = 100`, there shouldn't be these changes.